doc/user/content/sql/functions/extract.md

```diff
@@ -8,13 +8,15 @@ menu:
 
 `EXTRACT` returns some time component from a time-based value, such as the year from a Timestamp.
 
+See [`date_part`](../date-part) for the traditional Ingres equivalent function.
```

Comment:
It makes sense to point people from date_part to extract because extract is the new preferred way. Does it make sense to not even link people from extract to date_part, and instead remove this sentence?

Reply:
I removed this sentence.